srcpkgs/imwheel/template

```diff
@@ -0,0 +1,13 @@
+# Template file for 'imwheel'
+pkgname=imwheel
+version=1.0.0pre12
```

Comment:
Should probably be `1.0.0` and `pre12` appended where necessary, i.e. `wrksrc="${pkgname}-${version}pre12` and `distfiles="…${version}pre12.tar.gz`.
Otherwise the final `1.0.0` will be **smaller than** 1.0.0pre12 version wise and you'd have to add a `reverts=…`.
BTW Is there a reason not to wait for the stable release?

Reply:
Thanks for reviewing. 

If I change it like this I get 
=> ERROR: imwheel-1.0.0_1: cannot access wrksrc directory [/builddir/imwheel-1.0.0]
because /builddir/imwheel-1.0.0/imwheel-1.0.0pre12 is created.

This is the latest version available since years and also provided by other distributions. See e.g. https://www.archlinux.org/packages/community/x86_64/imwheel/

Reply:
That's why I set to add `wrksrc="${pkgname}-${version}pre12"` after the line `revision=1`.
Ok, so there will probably be no 1.0.0 in the forseeable future. Still, I'd go with the `pre12` separated out.

Reply:
If you install `xtools` you can `xlint srcpkgs/imwheel/template` locally to see what's missing or wrong.

Reply:
> That's why I suggested to add `wrksrc="${pkgname}-${version}pre12"` after the line `revision=1`.
Ah, I missed this one.